app/client/cypress/integration/national.spec.js

```diff
@@ -0,0 +1,188 @@
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+    cy.findByText(text).should('not.exist');
+  });
+
+  it('Clicking the Nutrient Pollution link opens a new tab', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'href', 'https://www.epa.gov/nutrientpollution');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'target', '_blank');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking the “Learn more about what EPA is doing to reduce nutrient pollution” link opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/nutrient-policy-data/what-epa-doing-reduce-nutrient-pollution',
+    );
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'target', '_blank');
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking on of the images in the “Learn more about water types” section opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results',
+    );
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should('have.attr', 'target', '_blank');
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should('have.attr', 'rel', 'noopener noreferrer');
```

Comment:
This one's a bit different... The user's really clicking an image (with a caption text) – they don't have any idea what the URL is, other than hovering over the linked image. The ideal way you'd select images  is with dom-testing-library's `findByAltText` query – that has the added benefit of making sure you're using alt text for images in the first place.

Here's a situation where I think it's ok to traverse the DOM w/ Cypress's `parent()` method a few times to get to the surrounding anchor. Normally you wouldn't want to do that, as it goes against the whole philosophy of the user not knowing or caring what the DOM structure looks like (and makes refactoring the structure of your markup break tests, which is not ideal). But in this case, it's probably the closest way to replicate what a user would be doing (looking at at an image, and clicking it). It could be argued that it'd be even better targeting the caption as well, but I'm good w/ this simpler approach:

```suggestion
    cy.findByAltText(/Learn more about Rivers & Streams/)
      .parent()
      .parent()
      .should(
        'have.attr',
        'href',
        'https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results',
      )
      .should('have.attr', 'target', '_blank')
      .should('have.attr', 'rel', 'noopener noreferrer');
```